SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -21,11 +21,54 @@ is_future_reef_map_file() {
     echo $file | egrep -q '^FutureReefMap_'
 }
 
-# main
-# $1 - file to handle
-main() {
+# handles a single netcdf file, return path in which file is stored
+# $1 - netcdf file
+handle_netcdf_file() {
+    local file=$1; shift
+    log_info "Handling SOOP CO2 file '$file'"
+
+    local recipient=`get_uploader_email $file`
+    if [ -n "$recipient" ]; then
+        echo "" | notify_by_email $recipient "Processing new underway CO2 file '$file'"
+    fi
+    echo "" | notify_by_email $BACKUP_RECIPIENT "Processing new underway CO2 file '$file'"
```

Comment:
The above 4 lines are repeated a few times in this script. Might be worth replacing defining a function to do it?